ui/raidboss/data/04-sb/eureka/eureka_hydatos.ts

```diff
@@ -249,6 +255,8 @@ const triggerSet: TriggerSet<Data> = {
       condition: (data, matches) => data.side === 'east' && data.me === matches.target,
       response: Responses.doritoStack(),
     },
+    // https://xivapi.com/PublicContentTextData/2122
+    // But we do not currently have a log line for this data.
```

Comment:
I haven't done this one so I can't try and track down the data source.

Reply:
> I haven't done this one so I can't try and track down the data source.

Based just on the endpoint (exd) name alone, it *looks* like it would be something close to `BattleTalk2`.  Looking through the derived csv, it looks like nearly all of it is Bozja/Eureka dialog, but some of the earliest (lowest id) rows are actually the dialog used in the Ceremony of Eternal Bonding.  Maybe next time you get a wedding invite...? 😄

Reply:
This looks to also be an `ActorControl` packet, I think. Still haven't done that side of BA yet, but an equivalent line from Art:
```
4096|2024-03-18T20:33:22.7130000-04:00|ActorControl|400058CA|2100|00000000|00000848|00000FA0|00000000|00000000|c862c35712ed4122
00|2024-03-18T20:33:22.0000000-04:00|0044|Art|O spear unyielding, become thee a thousand soldiers!|cf0c282daa64b90b
```

`0x848` = <https://xivapi.com/PublicContentTextData/2120>

Reply:
> This looks to also be an `ActorControl` packet, I think. Still haven't done that side of BA yet, but an equivalent line from Art:
> 
> ```
> 4096|2024-03-18T20:33:22.7130000-04:00|ActorControl|400058CA|2100|00000000|00000848|00000FA0|00000000|00000000|c862c35712ed4122
> 00|2024-03-18T20:33:22.0000000-04:00|0044|Art|O spear unyielding, become thee a thousand soldiers!|cf0c282daa64b90b
> ```
> 
> `0x848` = https://xivapi.com/PublicContentTextData/2120

It looks like `2100` didn't even make the list of `ActorControl` packet volume - which is unsurprising considering that `PublicContentTextData` seems to be largely (if not exclusively) limited to Eureka, Bozja/Zadnor, and weddings.  I'd be in favor of adding this to `ActorControlExtra` as well.  (Purely supposition, but this could also be some future-proofing for Cosmic Exploration?)